apps/monitoring/src/graphql-provider.tsx

```diff
@@ -1,7 +1,43 @@
 import { useAppBridge } from "@saleor/app-sdk/app-bridge";
-import { createGraphQLClient } from "@saleor/apps-shared";
 import { PropsWithChildren } from "react";
-import { Provider } from "urql";
+import { cacheExchange, createClient as urqlCreateClient, fetchExchange, Provider } from "urql";
+import { authExchange } from "@urql/exchange-auth";
+
+const createGraphQLClient = ({
```

Comment:
Why do we have to create client this way? I would appreciate comment since I'll be refactoring gql in the future.
If possible, would consider extending component from shared to minimize mentioned refactoring in the future.